### PR TITLE
Resolve 4150 commit issue

### PR DESCRIPTION
**Pull request by bpicoCode**
### Description
- Resolving Commit Issue (hopefully)

### Type of Change
- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by bpicoCode** [comment]
@daksh10110 Let me know how this looks.

**Review by dakshdeepHERE — APPROVED** [review]
This looks good @bpicoCode 🚀

File: content/javascript/concepts/dates/terms/getUTCHours/getUTCHours.md [hunk 1]
```diff
@@ -35,3 +35,15 @@ Returns the following to the console:
 ```shell
 2
 ```
+
+## Codebyte Example
+
+In the following example, variables `estDate` and `pstDate` are defined to store a new `Date` object in Eastern Time and Pacific Time respectively. The method `.getUTCHours()` is then used to print the hours of the dates according to universal time. You can run this example in the Codebyte below to test the results:
```
Comment (ishg-153) [inline content/javascript/concepts/dates/terms/getUTCHours/getUTCHours.md:41]:
```suggestion
In the following example, variables `estDate` and `pstDate` are defined to store a new `Date` object in Eastern Time and Pacific Time respectively. The method `.getUTCHours()` is then used to print the hours of the dates according to universal time. Run the example in the Codebyte below to test the results:
```
We avoid the use of personal pronouns in entries.

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@bpicoCode One small change from my end and then it's ready to go.

**Comment by bpicoCode** [comment]
> @bpicoCode One small change from my end and then it's ready to go.

Made the change!

**Pull request merged**